tests/scripts/all.sh

```diff
@@ -41,6 +41,8 @@ FORCE=0
 : ${GNUTLS_LEGACY_CLI:="$GNUTLS_CLI"}
 : ${GNUTLS_LEGACY_SERV:="$GNUTLS_SERV"}
 : ${OUT_OF_SOURCE_DIR:=./mbedtls_out_of_source_build}
+: ${ARMC5_BIN_DIR:=/usr/bin}
+: ${ARMC6_BIN_DIR:=/usr/bin}
```

Comment:
You're looking for the path to the compilers, and defaulting to /usr/bin.

I would suggest you find the full path of `armcc` and `armclang` using `which`, then extract the directory using `dirname`. (Or do something equivalent).